packages/react-reconciler/src/ReactFiber.js

```diff
@@ -139,6 +139,8 @@ export type Fiber = {|
   firstEffect: Fiber | null,
   lastEffect: Fiber | null,
 
+  thrownValue: any,
```

Comment:
You'll have to make the case for this addition.

Reply:
Strongest case I think is resuming. We should be able to reuse parts of the effect list during a bail out, including any of the values that were thrown.

Can you think of some viable alternatives? I'm having trouble coming up with one.